lib/ultragrep.rb

```diff
@@ -187,6 +187,10 @@ def ultragrep(options)
       lua = config.types[file_type]["lua"]
       collector = Ultragrep::LogCollector.new(config.log_path_glob(file_type), options)
       file_lists = collector.collect_files
+      if !file_lists
+        $stderr.puts("No log files found in date range #{Time.at(options.fetch(:range_start))} -- #{Time.at(options.fetch(:range_end))}")
```

Comment:
Based on the options parsing, it should be impossible for `:range_end` to be undefined. https://github.com/zendesk/ultragrep/blob/ahart/empty-log-error-addition/lib/ultragrep.rb#L95